lib/modules/download/generators/gdb.rb

```diff
@@ -19,17 +19,14 @@ def initialize zip_path, wdpa_ids
   def generate
     return false if @wdpa_ids.is_a?(Array) && @wdpa_ids.empty?
 
-    gdb_paths = []
-
     clean_up_after do
       QUERY_CONDITIONS.each do |name, props|
```

Comment:
Freeze `QUERY_CONDITIONS`?

Reply:
Well spotted. Done for both gdb and shapefile, thanks!